diff --git a/components/Tldraw.vue b/components/Tldraw.vue
index 561e10799ec47edb9542a5a3a0f32035827875e2..3df0c9f102059690d6d8a227f2adc64e1a136c9d 100644
--- a/components/Tldraw.vue
+++ b/components/Tldraw.vue
@@ -93,7 +93,7 @@ const state = reactive<State>({
 const store = useStore();
 
 const fetchSnapshot = async (doc: string) => {
-  const res = await fetch("/" + doc);
+  const res = await fetch("./" + doc);
   const snapshot = await res.json();
   loadSnapshot(store, snapshot);
 
